capture_gui/plugins/viewportplugin.py

```diff
@@ -203,31 +174,33 @@ def apply_inputs(self, inputs):
     def get_outputs(self):
         """
         Retrieve all settings of each available sub widgets
-        :return: 
+        
+        :return: collection of output settings for the viewport 
+        :rtype: dict
         """
 
         outputs = dict()
 
         high_quality = self.high_quality.isChecked()
         override_viewport_options = self.override_viewport.isChecked()
 
-        outputs['viewport2_options'] = dict()
-        outputs['viewport_options'] = dict()
+        if override_viewport_options:
+            outputs['viewport2_options'] = dict()
+            outputs['viewport_options'] = dict()
+            outputs['display_options'] = lib.get_background_colors()
```

Comment:
This seems odd when `override_viewport_options = True` because it doesn't actually override a setting but just takes it from the viewport. Are you sure this was your intended behavior?